README.md

```diff
@@ -17,47 +19,114 @@ Currently, we don't typically assign contributors. If you see an open issue that
 
 ## Setting Up
 
-1. Clone the repo
+### 1. Clone the repo
 
 ```
 git clone https://github.com/PolicyEngine/policyengine-api.git
 ```
 To contribute, clone the repository instead of forking it and then request to be added as a contributor.
 
-2. Activate your virtual environment
+### 2. Activate your virtual environment
 
-3. Install dependencies
+### 3. Install dependencies
 
 ```
 make install
 ```
 
-4. Start a server on localhost to see your changes
+### 4. Start a server on localhost to see your changes
+
+In api.py, comment out
```

Comment:
I think we can actually remove the CORS instructions, since the CORS errors were occurring due to Apple bluetooth jacking port 5000